### ISSUE TITLE
Some questions about class module and train

### ISSUE THREAD

**Issue by wangqiim** [Xiangtaokong/ClassSR#40]
Thanks, it is a interesting work. I have some questions to ask. Looking forward to your answer.
(1) You propose the Average-Loss and Class-Loss to train class module, It seems the class module will be trained to output the class result simple:medium:hard with 1:1:1. But in your paper, it is not.
(2) After trained the Class Module independently, why train Class & SR Module jointly? I think maybe the pre-trained SR Module has been the best?
(3) how many 2080Ti GPUS need to train(and total memory)? About how long it takes to train?

**Comment by Xiangtaokong** [comment]
Thanks for your attention.
(1) The class results depend on training data and test data. A sub-image is assigned to “simple” which means that the sub-image is similar to one-third of the simpler sub-images among training data. If test data contains more "simple" sub-images, the simple class results would be more than others.
(2) You can try it yourself~ Actually, the performance is still increasing while FLOPs is decreasing during joint training.
(3) For ClassSR-FSRCNN, two 2080Ti GPUs and about 3 days are enough.

Many operations do not need to strictly follow the paper, but can be modified according to different situations. (e.g. the rate of average-loss is also can be 25% : 25% : 50%)

**Comment by wangqiim** [comment]
> 

make sense. Thank you very much!
